scapy/contrib/gtp.uts

```diff
@@ -5,27 +5,31 @@
 
 + GTPv1
 
+= GTPHeader, basic instanciation
+
+a = GTPHeader()
+assert a.version == 1
+assert a.E == a.S == a.PN == 0
+
 = GTPCreatePDPContextRequest(), basic instanciation
 gtp = IP()/UDP(dport=2123)/GTPHeader(teid=2807)/GTPCreatePDPContextRequest()
-gtp.dport == 2123 and gtp.teid == 2807 and len(gtp.IE_list) == 5
+assert gtp.dport == 2123 and gtp.teid == 2807 and len(gtp.IE_list) == 5
+assert gtp[GTPCreatePDPContextRequest].next_ex == 0
 
 = GTPCreatePDPContextRequest(), basic dissection
 random.seed(0x2807)
-str(gtp) == b"E\x00\x00O\x00\x01\x00\x00@\x11|\x9b\x7f\x00\x00\x01\x7f\x00\x00\x01\x08K\x08K\x00;{N2\x10\x00+\x00\x00\n\xf7\xd2y\x00\x00\x10\xf8>\x14\x05\x14\t\x85\x00\x04\xa6A\xd8+\x85\x00\x04z\xafnt\x87\x00\x0fxKbPaePK9oq0pb5"
+str(gtp) == b'E\x00\x00O\x00\x01\x00\x00@\x11|\x9b\x7f\x00\x00\x01\x7f\x00\x00\x01\x08K\x08K\x00;}N0\x10\x00+\x00\x00\n\xf7\xd2y\x00\x00\x10\xf8>\x14\x05\x14\t\x85\x00\x04\xa6A\xd8+\x85\x00\x04z\xafnt\x87\x00\x0fxKbPaePK9oq0pb5'
 
 = GTPV1UpdatePDPContextRequest(), dissect
-h = "3333333333332222222222228100a38408004588006800000000fd1134820a2a00010a2a00024aa5084b005408bb32120044ed99aea9386f0000100000530514058500040a2a00018500040a2a000187000c0213921f739680fe74f2ffff94000130970001019800080112f41004d204d29900024000b6000101"
```

Comment:
Hi @gpotter2 and @guedou,

About the missing fields, our idea was : a message is made of the GTP-C header + the specific fields (for example a "create pdp context request" = GTPHeader / GTPUpdatePDPContextRequest). Thus the fields seq, npdu, and next extentions are inside the request/response classes (GTPUpdatePDPContextRequest for instance) and are not missing :-)                                                                                                                                                                                                                                                            
It would be a nice improvement to have a single header class (for GTP-U and GTP-C) and to move the basic elements in it. But then you have to remove these fields (seq, npdu and next_ex) from the requests and responses classes.                                                                                                                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                 
Why you did remove the check on the GTP version (line 199) ?